configs/sst_gpu-unwanted.yaml

```diff
@@ -31,6 +31,8 @@ data:
   - imake
   # Only used by optional plugin of hwloc, removed for RHEL.
   - libXNVCtrl
+  # Deprecated since RHEL 9 (technically even 8.4), removed in RHEL 10
+  - motif
```

Comment:
Since motif is still in RHEL 9 (even though it's deprecated), you'll want to split the unwanted config too

Reply:
oh, good catch, I hadn't thought of that! Should be fixed now :-)